decidim-admin/app/assets/javascripts/decidim/admin/external_domain_whitelist.js.es6

```diff
@@ -0,0 +1,97 @@
+((exports) => {
```

Comment:
I believe this file is missing from the precompile pipeline?

As this is included using the `javascript_include_tag`, we should add it to the admin JS manifest file.